packages/IMAPClient-Core.package/ICEndPoint.class/instance/fetchNextResponse.st

```diff
@@ -1,5 +1,6 @@
 fetching
 fetchNextResponse
+
```

Comment:
Is there a reason for all this whitespaces? :D 
- fetchNextResponse
- folderFromResponse
- getCurrentUIDs
- hierarchyDelimiter
- hierarchyDelimiter : aSymbol
- parseHeaderChunk.for
and some more. 

Maybe we should fine a consistant way to use whitespace in our project. Something like SWA-Lint could  help us.

Reply:
Also very much agreed. We should sit down once and establish some coding style conventions for our team.